### PR TITLE
add method

### PR DESCRIPTION
**Pull request by allenjeong**
added add_account and update parameters for resource

### CODE REVIEW

File: pypmp/pypmp.py [hunk 1]
```diff
@@ -347,3 +357,32 @@ def delete_user_by_username(self, username):
...
+    def add_account(
+        self, 
+        username, 
+        password, 
+        resourceid,
+        accountpasswordpolicy="Strong", 
+        notes=""
+        ):
+        if not notes:
+            notes = ""
```
Comment (pschmitt) [inline pypmp/pypmp.py:372]: I don't believe this check is necessary given that the default value is already above (line 369)

File: README-account.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Add account into existing Resource
+
+init branch
```
Comment (pschmitt) [inline README-account.md:3]: I think we can remove this :)

File: .gitignore [hunk 3]
```diff
@@ -111,3 +111,5 @@ dmypy.json
 
 Pipfile.lock
 .envrc
+.vscode/settings.json
+.gitignore
```
Comment (pschmitt) [inline .gitignore:115]: That's a bit redundant. And possibly dangerous.

```suggestion

```

File: README-account.md (hunk 2 above)
Comment (pschmitt) [reply README-account.md:3]:
```suggestion

```

File: pypmp/pypmp.py (hunk 1 above)
Comment (pschmitt) [reply pypmp/pypmp.py:372]:
```suggestion
```

**Review by pschmitt — CHANGES_REQUESTED** [review]
There's a few things I'd like to see removed from here. Other than that: LGTM. Thanks!

**Comment by allenjeong** [comment]
thanks for merging

**Pull request closed without merging**